tsconfig paths not respected

<!--
IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION YOUR ISSUE MIGHT BE CLOSED WITHOUT INVESTIGATING
-->
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
<!--
Output from: `ng --version`.
If nothing, output from: `node --version` and `npm --version`.
  Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra?)
-->
```
@angular/cli: 1.4.8
node: 8.7.0
os: linux x64
```

### Repro steps.
<!--
Simple steps to reproduce this bug.
Please include: commands run, packages added, related code changes.
A link to a sample repo would help too.
-->
1. `ng serve` https://github.com/buu700/ngcli-paths

2. Either "success" or "failure" will be displayed.

### Desired functionality.
<!--
What would like to see implemented?
What is the usecase?
-->
tsconfig paths should be treated as equivalent to `resolve.alias` values, as far as I understand. However, it seems that they're only applying to imports from within the Angular project itself, not third-party modules. (I'd thought that this worked in the past, so it may be a recent regression.)

In this example, we're importing a node module `test63987` that imports and re-exports an arbitrary second module (`libsodium-wrappers-sumo`). Setting `"libsodium-wrappers-sumo": ["libsodium-replacement"]` in tsconfig does not have the desired effect. However, ejecting and adding the equivalent `alias` to webpack.config.js does work as expected.

### Mention any other details that might be useful.
<!-- Please include a link to the repo if this is related to an OSS project. -->

Project this affects is https://github.com/cyph/cyph.

---

Edit: **Workaround:** Install the latest Angular/CLI/etc. and replace `node_modules/@ngtools/webpack/src/paths-plugin.js` with https://github.com/buu700/ngtools-webpack-tmp/blob/master/src/paths-plugin.js.

@filipesilva here is my repro:

### Versions.
<!--
Output from: `ng --version`.
If nothing, output from: `node --version` and `npm --version`.
  Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra?)
-->
```
    _                      _                 ____ _     ___
   / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
  / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
 / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
/_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
               |___/
@angular/cli: 1.3.0
node: 6.9.2
os: darwin x64
@angular/animations: 4.4.3
@angular/common: 4.4.3
@angular/compiler: 4.4.3
@angular/core: 4.4.3
@angular/forms: 4.4.3
@angular/http: 4.4.3
@angular/platform-browser: 4.4.3
@angular/platform-browser-dynamic: 4.4.3
@angular/router: 4.4.3
@angular/cli: 1.2.7
@angular/compiler-cli: 4.4.3
@angular/language-service: 4.4.3

MacOS OS HighSierra 10.
```

### Repro steps.
<!--
Simple steps to reproduce this bug.
Please include: commands run, packages added, related code changes.
A link to a sample repo would help too.
-->
1. Pull my repro: https://github.com/glebmachine/angular2-3rdparty-resolve-issue
2. `yarn; ng build`
3. You've got resolving error
4. Replace angular/cli version to 1.2.7
5. `yarn; ng build`
6. Everything works great

### The log given by the failure.
<!-- Normally this include a stack trace and some more information. -->
```

ERROR in ./node_modules/greensock/src/TimelineMax.js
Module not found: Error: Can't resolve 'TweenLite' in '/Users/glebmikheev/angular-scrollmagic/node_modules/greensock/src'
resolve 'TweenLite' in '/Users/glebmikheev/angular-scrollmagic/node_modules/greensock/src'
  Parsed request is a module
  using description file: /Users/glebmikheev/angular-scrollmagic/node_modules/greensock/package.json (relative path: ./src)
  after using description file: /Users/glebmikheev/angular-scrollmagic/node_modules/greensock/package.json (relative path: ./src)
    resolve as module
      /Users/glebmikheev/angular-scrollmagic/node_modules/greensock/src/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/greensock/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/glebmikheev/angular-scrollmagic/node_modules
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules/TweenLite)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite doesn't exist
      looking for modules in /Users/glebmikheev/node_modules
        No description file found
        No description file found
        no extension
          /Users/glebmikheev/node_modules/TweenLite doesn't exist
        .ts
          /Users/glebmikheev/node_modules/TweenLite.ts doesn't exist
        .js
          /Users/glebmikheev/node_modules/TweenLite.js doesn't exist
        as directory
          /Users/glebmikheev/node_modules/TweenLite doesn't exist
      looking for modules in /Users/glebmikheev/angular-scrollmagic/node_modules
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules/TweenLite)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite doesn't exist
      looking for modules in /Users/glebmikheev/angular-scrollmagic/src
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src/TweenLite)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenLite doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenLite.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenLite.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/src/TweenLite doesn't exist
      looking for modules in /Users/glebmikheev/angular-scrollmagic/src
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src/TweenLite)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenLite doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenLite.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenLite.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/src/TweenLite doesn't exist
[/Users/glebmikheev/angular-scrollmagic/node_modules/greensock/src/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/greensock/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/node_modules]
[/Users/node_modules]
[/node_modules]
[/Users/glebmikheev/node_modules/package.json]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite.ts]
[/Users/glebmikheev/node_modules/TweenLite/package.json]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite.js]
[/Users/glebmikheev/node_modules/TweenLite]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite]
[/Users/glebmikheev/node_modules/TweenLite.ts]
[/Users/glebmikheev/node_modules/TweenLite.js]
[/Users/glebmikheev/node_modules/TweenLite]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite.ts]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite.js]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenLite]
[/Users/glebmikheev/angular-scrollmagic/src/TweenLite]
[/Users/glebmikheev/angular-scrollmagic/src/TweenLite.ts]
[/Users/glebmikheev/angular-scrollmagic/src/TweenLite.js]
[/Users/glebmikheev/angular-scrollmagic/src/TweenLite]
[/Users/glebmikheev/angular-scrollmagic/src/TweenLite]
[/Users/glebmikheev/angular-scrollmagic/src/TweenLite.ts]
[/Users/glebmikheev/angular-scrollmagic/src/TweenLite.js]
[/Users/glebmikheev/angular-scrollmagic/src/TweenLite]
 @ ./node_modules/greensock/src/TimelineMax.js 1298:2-34
 @ ./src/app/app.component.ts
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi ./src/main.ts
ERROR in ./node_modules/scrollmagic/scrollmagic/uncompressed/plugins/animation.gsap.js
Module not found: Error: Can't resolve 'TweenMax' in '/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/plugins'
resolve 'TweenMax' in '/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/plugins'
  Parsed request is a module
  using description file: /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/package.json (relative path: ./scrollmagic/uncompressed/plugins)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/package.json (relative path: ./scrollmagic/uncompressed/plugins)
    resolve as module
      /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/plugins/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/glebmikheev/angular-scrollmagic/node_modules
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules/TweenMax)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax doesn't exist
      looking for modules in /Users/glebmikheev/node_modules
        No description file found
        Field 'browser' doesn't contain a valid alias configuration
        No description file found
        no extension
          Field 'browser' doesn't contain a valid alias configuration
          /Users/glebmikheev/node_modules/TweenMax doesn't exist
        .ts
          Field 'browser' doesn't contain a valid alias configuration
          /Users/glebmikheev/node_modules/TweenMax.ts doesn't exist
        .js
          Field 'browser' doesn't contain a valid alias configuration
          /Users/glebmikheev/node_modules/TweenMax.js doesn't exist
        as directory
          /Users/glebmikheev/node_modules/TweenMax doesn't exist
      looking for modules in /Users/glebmikheev/angular-scrollmagic/node_modules
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules/TweenMax)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax doesn't exist
      looking for modules in /Users/glebmikheev/angular-scrollmagic/src
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src/TweenMax)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenMax doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenMax.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenMax.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/src/TweenMax doesn't exist
      looking for modules in /Users/glebmikheev/angular-scrollmagic/src
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src/TweenMax)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenMax doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenMax.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TweenMax.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/src/TweenMax doesn't exist
[/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/plugins/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/node_modules]
[/Users/node_modules]
[/node_modules]
[/Users/glebmikheev/node_modules/package.json]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax.ts]
[/Users/glebmikheev/node_modules/TweenMax/package.json]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax.js]
[/Users/glebmikheev/node_modules/TweenMax]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax]
[/Users/glebmikheev/node_modules/TweenMax.ts]
[/Users/glebmikheev/node_modules/TweenMax.js]
[/Users/glebmikheev/node_modules/TweenMax]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax.ts]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax.js]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TweenMax]
[/Users/glebmikheev/angular-scrollmagic/src/TweenMax]
[/Users/glebmikheev/angular-scrollmagic/src/TweenMax.ts]
[/Users/glebmikheev/angular-scrollmagic/src/TweenMax.js]
[/Users/glebmikheev/angular-scrollmagic/src/TweenMax]
[/Users/glebmikheev/angular-scrollmagic/src/TweenMax]
[/Users/glebmikheev/angular-scrollmagic/src/TweenMax.ts]
[/Users/glebmikheev/angular-scrollmagic/src/TweenMax.js]
[/Users/glebmikheev/angular-scrollmagic/src/TweenMax]
 @ ./node_modules/scrollmagic/scrollmagic/uncompressed/plugins/animation.gsap.js 31:2-61
 @ ./src/app/extensions/scrollmagic-helpers.ts
 @ ./src/app/app.component.ts
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi ./src/main.ts
ERROR in ./node_modules/scrollmagic/scrollmagic/uncompressed/plugins/animation.gsap.js
Module not found: Error: Can't resolve 'TimelineMax' in '/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/plugins'
resolve 'TimelineMax' in '/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/plugins'
  Parsed request is a module
  using description file: /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/package.json (relative path: ./scrollmagic/uncompressed/plugins)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/package.json (relative path: ./scrollmagic/uncompressed/plugins)
    resolve as module
      /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/plugins/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/node_modules doesn't exist or is not a directory
      /Users/glebmikheev/angular-scrollmagic/node_modules/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/glebmikheev/angular-scrollmagic/node_modules
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules/TimelineMax)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax doesn't exist
      looking for modules in /Users/glebmikheev/node_modules
        No description file found
        Field 'browser' doesn't contain a valid alias configuration
        No description file found
        no extension
          Field 'browser' doesn't contain a valid alias configuration
          /Users/glebmikheev/node_modules/TimelineMax doesn't exist
        .ts
          Field 'browser' doesn't contain a valid alias configuration
          /Users/glebmikheev/node_modules/TimelineMax.ts doesn't exist
        .js
          Field 'browser' doesn't contain a valid alias configuration
          /Users/glebmikheev/node_modules/TimelineMax.js doesn't exist
        as directory
          /Users/glebmikheev/node_modules/TimelineMax doesn't exist
      looking for modules in /Users/glebmikheev/angular-scrollmagic/node_modules
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./node_modules/TimelineMax)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax doesn't exist
      looking for modules in /Users/glebmikheev/angular-scrollmagic/src
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src/TimelineMax)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TimelineMax doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TimelineMax.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TimelineMax.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/src/TimelineMax doesn't exist
      looking for modules in /Users/glebmikheev/angular-scrollmagic/src
        using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src)
          using description file: /Users/glebmikheev/angular-scrollmagic/package.json (relative path: ./src/TimelineMax)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TimelineMax doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TimelineMax.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/glebmikheev/angular-scrollmagic/src/TimelineMax.js doesn't exist
            as directory
              /Users/glebmikheev/angular-scrollmagic/src/TimelineMax doesn't exist
[/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/plugins/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/uncompressed/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/scrollmagic/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/scrollmagic/node_modules]
[/Users/glebmikheev/angular-scrollmagic/node_modules/node_modules]
[/Users/node_modules]
[/node_modules]
[/Users/glebmikheev/node_modules/package.json]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax.ts]
[/Users/glebmikheev/node_modules/TimelineMax/package.json]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax.js]
[/Users/glebmikheev/node_modules/TimelineMax]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax]
[/Users/glebmikheev/node_modules/TimelineMax.ts]
[/Users/glebmikheev/node_modules/TimelineMax.js]
[/Users/glebmikheev/node_modules/TimelineMax]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax.ts]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax.js]
[/Users/glebmikheev/angular-scrollmagic/node_modules/TimelineMax]
[/Users/glebmikheev/angular-scrollmagic/src/TimelineMax]
[/Users/glebmikheev/angular-scrollmagic/src/TimelineMax.ts]
[/Users/glebmikheev/angular-scrollmagic/src/TimelineMax.js]
[/Users/glebmikheev/angular-scrollmagic/src/TimelineMax]
[/Users/glebmikheev/angular-scrollmagic/src/TimelineMax]
[/Users/glebmikheev/angular-scrollmagic/src/TimelineMax.ts]
[/Users/glebmikheev/angular-scrollmagic/src/TimelineMax.js]
[/Users/glebmikheev/angular-scrollmagic/src/TimelineMax]
 @ ./node_modules/scrollmagic/scrollmagic/uncompressed/plugins/animation.gsap.js 31:2-61
 @ ./src/app/extensions/scrollmagic-helpers.ts
 @ ./src/app/app.component.ts
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi ./src/main.ts
```

--- a/packages/@ngtools/webpack/src/paths-plugin.ts
+++ b/packages/@ngtools/webpack/src/paths-plugin.ts
@@ -124,7 +124,7 @@ export class PathsPlugin implements Tapable {
     this._nmf.plugin('before-resolve', (request: NormalModuleFactoryRequest,
                                         callback: Callback<any>) => {
       // Only work on TypeScript issuers.
-      if (!request.contextInfo.issuer || !request.contextInfo.issuer.endsWith('.ts')) {
+      if (!request.contextInfo.issuer || !request.contextInfo.issuer.match(/\.[jt]s$/)) {
         return callback(null, request);
       }
 

